README.md

```diff
@@ -31,467 +32,48 @@ Usage
-        - store_artifacts:
-            path: wheelhouse/
-
-    osx-wheels:
-      working_directory: ~/osx-wheels
-      macos:
-        xcode: "10.0.0"
-      steps:
-        - checkout
-        - run:
-            name: Build the OS X wheels.
-            command: |
-              pip install --user cibuildwheel
-              cibuildwheel --output-dir wheelhouse
-        - store_artifacts:
-            path: wheelhouse/
-
-  workflows:
-    version: 2
-    all-tests:
-      jobs:
-        - linux-wheels
-        - osx-wheels
-  ```
-
-  Note: CircleCI doesn't enable free macOS containers for open source by default, but you can ask for access. See [here](https://circleci.com/docs/2.0/oss/#overview) for more information.
-
-  CircleCI will store the built wheels for you - you can access them from the project console.
-
-</details>
-
-
-<details>
-    <summary><b>AppVeyor</b>
-        <img width="16" src="https://unpkg.com/simple-icons@latest/icons/windows.svg" />
-    </summary>
-
-- To build Windows wheels on AppVeyor, create an `appveyor.yml` file in your repo.
-
-    ```
-    build_script:
-      - pip install cibuildwheel==0.12.0
-      - cibuildwheel --output-dir wheelhouse
-    artifacts:
-      - path: "wheelhouse\\*.whl"
-        name: Wheels
-    ```
-    
-  AppVeyor will store the built wheels for you - you can access them from the project console. Alternatively, you may want to store them in the same place as the Travis CI build. See [AppVeyor deployment docs](https://www.appveyor.com/docs/deployment/) for more info, or see [Delivering to PyPI](#delivering-to-pypi) below.
-    
-</details>
-
-- Commit those files, enable building of your repo on Travis CI and AppVeyor, and push.
-
-All being well, you should get wheels delivered to you in a few minutes. 
-
-> ⚠️ Got an error? Check the [checklist](#it-didnt-work) below.
-
-### Configuration overview
-
-`cibuildwheel` allows for easy customization of the various phases of the build process demonstrated above:
-
-|   | Option |   |
-|---|---|---|
-| **Target wheels** | `CIBW_PLATFORM` | Override the auto-detected target platform |
-|   | `CIBW_BUILD` | Build only certain Python versions |
-|   | `CIBW_SKIP` | Skip certain Python versions |
-| **Build parameters** | `CIBW_BUILD_VERBOSITY` | Increase or decrease the output of `pip wheel` |
-| **Build environment** | `CIBW_ENVIRONMENT` | Set environment variables needed during the build |
-|   | `CIBW_BEFORE_BUILD` | Execute a shell command preparing each wheel's build |
-|   | `CIBW_MANYLINUX1_X86_64_IMAGE` | Specify an alternative manylinx1 x86_64 docker image |
-|   | `CIBW_MANYLINUX1_I686_IMAGE` | Specify an alternative manylinux1 i686 docker image |
-| **Tests** | `CIBW_TEST_COMMAND` | Execute a shell command to test all built wheels |
-|   | `CIBW_TEST_REQUIRES` | Install Python dependencies before running the tests |
-|   | `CIBW_TEST_EXTRAS` | Install Python dependencies before running the tests using ``extras_require``|
```

Comment:
> But then again, we would need to keep README and docs in sync
it's possible to refer to portions of the README from the docs without having to maintain them in 2 places. sample in https://github.com/mayeut/pybase64/blob/master/docs/changelog.rst
This however prevents from having different layouts for these portions.

Reply:
Uuuh, that's interesting!
@joerick is using MkDocs and markdown, though. Does it also work for that, do you know?

Reply:
I completely overlooked that fact when commenting... I don't know for those.

Reply:
I kinda agree, it would be nice to bring a table summarising the options into the readme, and have it link to the docs site. 

@mayeut  That would be ideal, but I don't think that's possible in mkdocs, unless we could find/write a plugin to do it..?

Reply:
Progress here- I've written a little JS that writes a markdown table that we can copy/paste into the readme, so we can keep them in sync- here's current output

|   | Option | Description |
|---|--------|-------------|
| **Build selection** | [`CIBW_PLATFORM`](https://cibuildwheel.readthedocs.io/en/stable/options/#platform)  | Override the auto-detected target platform |
|   | [`CIBW_BUILD`](https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip)  [`CIBW_SKIP`](https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip)  | Choose the Python versions to build |
| **Build environment** | [`CIBW_ENVIRONMENT`](https://cibuildwheel.readthedocs.io/en/stable/options/#environment)  | Set environment variables needed during the build |
|   | [`CIBW_BEFORE_BUILD`](https://cibuildwheel.readthedocs.io/en/stable/options/#before-build)  | Execute a shell command preparing each wheel's build |
|   | [`CIBW_MANYLINUX1_X86_64_IMAGE`](https://cibuildwheel.readthedocs.io/en/stable/options/#manylinux-image)  [`CIBW_MANYLINUX1_I686_IMAGE`](https://cibuildwheel.readthedocs.io/en/stable/options/#manylinux-image)  | Specify alternative manylinux1 x86_64 docker images |
| **Testing** | [`CIBW_TEST_COMMAND`](https://cibuildwheel.readthedocs.io/en/stable/options/#test-command)  | Execute a shell command to test all built wheels |
|   | [`CIBW_TEST_REQUIRES`](https://cibuildwheel.readthedocs.io/en/stable/options/#test-requires)  | Install Python dependencies before running the tests |
|   | [`CIBW_TEST_EXTRAS`](https://cibuildwheel.readthedocs.io/en/stable/options/#test-extras)  | Install your wheel for testing using extras_require |
| **Other** | [`CIBW_BUILD_VERBOSITY`](https://cibuildwheel.readthedocs.io/en/stable/options/#test-extras)  | Increase/decrease the output of pip wheel |

Reply:
(the above links will work once we do a release. in the meantime, in the URL replace 'stable' with 'docs')